source/getting-started/installation-raspberry-pi.markdown

```diff
@@ -1,29 +1,207 @@
 ---
 layout: page
-title: "Installation on a Raspberry Pi"
-description: "Instructions to install Home Assistant on a Raspberry Pi."
-date: 2016-04-16 11:36
+title: "Raspbian Lite on a Raspberry Pi"
+description: "Instructions to install Home Assistant on a Raspberry Pi runnning Raspbian Lite."
+date: 2016-09-05 16:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 ---
 
-Home Assistant requires the Raspberry Pi to run [Raspbian Jessie](https://www.raspberrypi.org/downloads/raspbian/). This version was released on September 24, 2015 and comes by default with Python 3.4 which is required for Home Assistant.
+### {% linkable_title Installation %}
 
-Execute the following code in a console:
+This installation of Home Assistant requires the Raspberry Pi to run [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/).
+The installation will be installed in a [Virtual Environment](/getting-started/installation-virtualenv) with minimal overhead. Instructions assume this is a new installation.
 
+Connect to the Raspberry Pi over ssh. Default password is `raspberry`.
+```bash
+$ ssh pi@ipadress
+```
+
+Changing the default password is encouraged.
+```bash
+$ passwd
+```
+
+Update the system.
+```bash
+$ sudo apt-get update
+$ sudo apt-get upgrade -y
+```
+
+Install the dependencies.
+```bash
+$ sudo apt-get install python3 python3-venv python3-pip python3-sphinx
+$ sudo pip3 install virtualenv
+```
+
+Add a account for Home Assistant called `homeassistant`.  
+Since this account is only for running Home Assistant the extra arguments of `-rm` is added to create a system account and create a home directory.
+```bash
+$ sudo useradd -rm homeassistant
+```
+
+Next we will create a directory for the installation of Home Assistant and change the owner to the `homeassistant` account.
+```bash
+$ cd /srv
+$ sudo mkdir homeassistant
+$ sudo chown homeassistant:homeassistant homeassistant
+```
+
+Next up is to create and change to a virtual environment for Home Assistant. This will be done as the `homeassistant` account.
+```bash
+$ su -s /bin/bash homeassistant 
+$ cd /srv/homeassistant
+$ virtualenv -p python3 homeassistant_venv
+$ source /srv/homeassistant/homeassistant_venv/bin/activate
```

Comment:
I would put a bit more emphasis on venv here, maybe something in this line
""
Once you have activate you virtual environment you will notice the prompt change and the you can install homeassistant
```bash
(homeassistant_venv) $ pip install homeassistant
```
""

(in my venv I always run python only, not python3)
I also had some issue with pip (but maybe I did not have all dependencies at the time like you prepared above, so I had to do this from my notes -- and this might be a drawback of venv compared to virtualenv:
```bash
python3 -m venv --without-pip venv
source ./venv/bin/activate
curl https://bootstrap.pypa.io/get-pip.py | python 3
```